src/js/references/components/__tests__/ReferenceList.test.tsx

```diff
@@ -134,15 +143,17 @@ describe("<ReferenceList />", () => {
         });
 
         it("should display an error when name input is cleared", async () => {
+            const permissions = createFakePermissions({ create_ref: true });
+            const account = createFakeAccount({ permissions: permissions });
+            mockAPIGetAccount(account);
             const scope = mockApiGetReferences([references]);
             renderWithRouter(<ReferenceList />, state, history);
 
             expect(await screen.findByText("References")).toBeInTheDocument();
+            expect(await screen.findByRole("link", { name: "clone" })).toBeInTheDocument();
             await userEvent.click(screen.getByRole("link", { name: "clone" }));
-
-            await userEvent.clear(screen.getByRole("textbox"));
             await userEvent.click(screen.getByRole("button", { name: "Clone" }));
-
+            await userEvent.clear(screen.getByRole("textbox"));
             expect(screen.getByText("Required Field")).toBeInTheDocument();
```

Comment:
Hmmmmm something seems strange here. In theory the textbox clear should have to come before the clone button is clicked. Otherwise I would expect the component happily accept the default name and never render an error. This combined with the other comment might make for a good new issue.

Reply:
You are right, the order of these two lines should be swapped. This was my mistake in the refactor. Good catch!